### ISSUE TITLE
Unsupported external module

### ISSUE THREAD

**Issue by wjagodfrey** [Snyk/snyk#16]
- `node -v`: 5.6.0
- `npm -v`: 3.6.0
- `snyk -v`: 1.11.2
- Command run:
`snyk test`

### Expected behaviour
Snyk tests should run, and give a clear success or failure (as they were yesterday)

### Actual behaviour
Snyk starts with the usual `Querying vulnerabilities database...`, before showing `not supported: external module: ikt@git+http://ikt.pm2.io/ikt.git#master` and exiting with exit code 1.

### Steps to reproduce
add `"pm2": "^1.0.1"` to your package.json, and run the above command, or `snyk wizard`.

While I realise this issue is with pm2, unless this is a vulnerability, I would like it to pass. It's an optional dependancy after all. (And if this is a vulnerability, have it fail as a vulnerability).

---

If applicable, please append the `--debug` flag on your command and include the output here **ensuring to remove any sensitive/personal details or tokens.

```
  snyk test +0ms { _: [ [Circular] ], debug: true }
  snyk checking for cli updates +391ms
  snyk add +26ms local true
  snyk add +799ms error Error: not supported: external module: ikt@git+http://ikt.pm2.io/ikt.git#master
    at supported (/usr/local/lib/node_modules/snyk/node_modules/snyk-module/lib/index.js:117:13)
    at moduleToObject (/usr/local/lib/node_modules/snyk/node_modules/snyk-module/lib/index.js:56:10)
    at pluck (/usr/local/lib/node_modules/snyk/node_modules/snyk-resolve-deps/lib/pluck.js:40:15)
    at walkDepsPicker (/usr/local/lib/node_modules/snyk/node_modules/snyk-resolve-deps/lib/logical.js:122:17)
    at Array.reduce (native)
    at walkDeps (/usr/local/lib/node_modules/snyk/node_modules/snyk-resolve-deps/lib/logical.js:118:28)
    at walkDepsPicker (/usr/local/lib/node_modules/snyk/node_modules/snyk-resolve-deps/lib/logical.js:146:28)
    at Array.reduce (native)
    at walkDeps (/usr/local/lib/node_modules/snyk/node_modules/snyk-resolve-deps/lib/logical.js:118:28)
    at logicalTree (/usr/local/lib/node_modules/snyk/node_modules/snyk-resolve-deps/lib/logical.js:40:30)
Error: not supported: external module: ikt@git+http://ikt.pm2.io/ikt.git#master
    at supported (/usr/local/lib/node_modules/snyk/node_modules/snyk-module/lib/index.js:117:13)
    at moduleToObject (/usr/local/lib/node_modules/snyk/node_modules/snyk-module/lib/index.js:56:10)
    at pluck (/usr/local/lib/node_modules/snyk/node_modules/snyk-resolve-deps/lib/pluck.js:40:15)
    at walkDepsPicker (/usr/local/lib/node_modules/snyk/node_modules/snyk-resolve-deps/lib/logical.js:122:17)
    at Array.reduce (native)
    at walkDeps (/usr/local/lib/node_modules/snyk/node_modules/snyk-resolve-deps/lib/logical.js:118:28)
    at walkDepsPicker (/usr/local/lib/node_modules/snyk/node_modules/snyk-resolve-deps/lib/logical.js:146:28)
    at Array.reduce (native)
    at walkDeps (/usr/local/lib/node_modules/snyk/node_modules/snyk-resolve-deps/lib/logical.js:118:28)
    at logicalTree (/usr/local/lib/node_modules/snyk/node_modules/snyk-resolve-deps/lib/logical.js:40:30)
  snyk { command: 'test', args: [ { debug: true } ], metadata: { local: true, error: 'Error: not supported: external module: ikt@git+http://ikt.pm2.io/ikt.git#master\n    at supported (/usr/local/lib/node_modules/snyk/node_modules/snyk-module/lib/index.js:117:13)\n    at moduleToObject (/usr/local/lib/node_modules/snyk/node_modules/snyk-module/lib/index.js:56:10)\n    at pluck (/usr/local/lib/node_modules/snyk/node_modules/snyk-resolve-deps/lib/pluck.js:40:15)\n    at walkDepsPicker (/usr/local/lib/node_modules/snyk/node_modules/snyk-resolve-deps/lib/logical.js:122:17)\n    at Array.reduce (native)\n    at walkDeps (/usr/local/lib/node_modules/snyk/node_modules/snyk-resolve-deps/lib/logical.js:118:28)\n    at walkDepsPicker (/usr/local/lib/node_modules/snyk/node_modules/snyk-resolve-deps/lib/logical.js:146:28)\n    at Array.reduce (native)\n    at walkDeps (/usr/local/lib/node_modules/snyk/node_modules/snyk-resolve-deps/lib/logical.js:118:28)\n    at logicalTree (/usr/local/lib/node_modules/snyk/node_modules/snyk-resolve-deps/lib/logical.js:40:30)' }, version: '1.11.1', os: 'OS X Yosemite', id: '306f365cf67ab797cddeb76a95c4befb76b53982', ci: false } +3ms
```

**Comment by remy** [comment]
Thanks for the report, I'm pretty sure I know where this issue is and I'll ping you when I've got a new release out (should be today).

**Comment by remy** [comment]
This isn't live yet, but the fix has landed in the core module. Will ping when it's live.

**Comment by remy** [comment]
Fixed and live in snyk@1.11.3. 

Thanks again.

**Comment by wjagodfrey** [comment]
Great work, thanks @remy! All sorted :ok_hand:
